inventory/byo/hosts.origin.example

```diff
@@ -95,6 +95,22 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # LDAP auth
 #openshift_master_identity_providers=[{'name': 'my_ldap_provider', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['dn'], 'email': ['mail'], 'name': ['cn'], 'preferredUsername': ['uid']}, 'bindDN': '', 'bindPassword': '', 'ca': '', 'insecure': 'false', 'url': 'ldap://ldap.example.com:389/ou=users,dc=example,dc=com?uid'}]
 
+# Cloud provider facts
+#
+# AWS
+#openshift_cloud_provider=aws
+#openshift_cloud_provider_aws_access_key=AWS_ACCESS_KEY_ID
+#openshift_cloud_provider_aws_secret_key=AWS_SECRET_ACCESS_KEY
```

Comment:
We'll probably want to document how to set these values using the lookup('env') so that users *can* use their environment variables if they wish.